packages/perseus/src/widgets/sorter/sorter.tsx

```diff
@@ -42,15 +42,8 @@ class Sorter extends React.Component<Props, State> {
         linterContext: linterContextDefault,
     };
 
-    static validate(state: any, rubric: any): any {
-        const correct = _.isEqual(state.options, rubric.correct);
-
-        return {
-            type: "points",
-            earned: correct ? 1 : 0,
-            total: 1,
-            message: null,
-        };
+    static validate(userInput: UserInput, rubric: Rubric): PerseusScore {
```

Comment:
Let me know if I should leave the old types here. My brain was having trouble wrapping around the older types, so I just fixed it.

Reply:
Nope, this is good how you've done this. We want to trend to stronger typing where possible (especially migrating away from `any` whenever we can). So I'm a 👍  to this.

Reply:
+1, we should always be fixing types if we think we can find a simple solution.